src/main/java/de/rabea/player/UnbeatableComputerPlayer.java

```diff
@@ -12,23 +12,23 @@ public UnbeatableComputerPlayer(Mark mark) {
 
     @Override
     public int getPosition(Board board) {
-        int remainingMovesCount = board.emptyCells().size();
-        return minimax(remainingMovesCount, Integer.MIN_VALUE, Integer.MAX_VALUE, board, mark).getMove();
+        int MAXIMUM_DEPTH = 8;
```

Comment:
Idiomatic Java has camelCase for local variables and UPPER_CASE for constants (`static final`).